{"uuid": "f7c0e92e-3598-4c43-a5de-26f5a32cf946", "children": ["ed20940a-78c9-4ab3-b78e-1d1a0dd54f58"], "befores": [{"name": "setup_login_uno", "status": "passed", "start": 1751938019856, "stop": 1751938045190}], "afters": [{"name": "setup_login_uno::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: Tried to run command without establishing a connection\n\n", "trace": "  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\PYTEST\\Fixture_decorate2.py\", line 41, in setup_login_uno\n    driver.close()\n    ~~~~~~~~~~~~^^\n  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 579, in close\n    self.execute(Command.CLOSE)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\saul.frias\\PycharmProjects\\SeleniumProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1751938081506, "stop": 1751938081509}], "start": 1751938019856, "stop": 1751938081509}